score/requirements.txt

```diff
@@ -1,6 +1,76 @@
-ipython
-jupyter
-jupyter_contrib_nbextensions
-numpy
-pandas
-requests
+appnope==0.1.2; sys_platform == "darwin" and python_version >= "3.7" and platform_system == "Darwin"
```

Comment:
Hmm, I'm guessing there's actually all the "downstream" dependencies here. (E.g., this file could also just have 10 dependencies in it that would install their own pre-requisites.) It might be a little cleaner to only list the "upstream" dependencies. 

But! I don't know `Poetry` at all, so if this is standard and the easiest way to generate it, that's fine.

Reply:
This is handled by `Poetry` and not meant for human consumption, just for CI/CD consumption. Which is the advantage of using `Poetry` in the first place.